build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "2.0.6"
+    version = "2.0.7"
```

Comment:
I normally use this  http://semver.org/ as a basis for versioning. 

-This hasn't introduced any breaking changes so it is not a major.
-This hasn't introduced any new functionality that is visible to the client. You could argue that an update to ExoPlayer could make this a minor but we haven't provided a way for a client to access these new changes.
- Patch is normally used for bug fixes.